factory/consensus/consensusComponents.go

```diff
@@ -667,6 +672,19 @@ func (ccf *consensusComponentsFactory) createPeerBlacklistHandler() (consensus.P
 	return blacklist.NewPeerBlacklist(blacklistArgs)
 }
 
+func (ccf *consensusComponentsFactory) createP2pSigningHandler() (consensus.P2PSigningHandler, error) {
+	p2pSignerArgs := &p2pFactory.ArgsMessageVerifier{
+		Marshaller: ccf.coreComponents.InternalMarshalizer(),
+		P2PSigner:  ccf.networkComponents.NetworkMessenger(),
+	}
+	p2pSigningHandler, err := p2pFactory.NewMessageVerifier(*p2pSignerArgs)
```

Comment:
You can replace L680-L685 with: `return p2pFactory.NewMessageVerifier(p2pSignerArgs)`

Reply:
done